### DISCUSSION TITLE
SQL Varchar allocation

### DISCUSSION THREAD

**Discussion by rafaelmag110** [eclipse-tractusx/tractusx-edc#1142] [Q&A]
Hi community,

I was wondering if there is any specific reason for the VARCHAR allocation in the sql init statements to be fixed to 255.
In the upstream implementation there seems to be no length limitation for these attributes. There is also no validation on the DMA during POST or PUT actions, which might lead to side effects as described in this open issue: https://github.com/eclipse-tractusx/tractusx-edc/issues/1127

What could be the right approach to fix this issue here? Changing the Varchar allocation size or adding validation to the DMA? The latter seems unreasonable since different implementations of the upstream might have different requirements, but one never knows...

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
